src/api/spec/controllers/webui/main_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Webui::MainController do
+  let(:user) { create(:confirmed_user) }
```

Comment:
You can even see that from the successful test run in travis. If there would be a cassete missing, it would fail because of that.